iml-agent/src/action_plugins/stratagem/action_filesync.rs

```diff
@@ -165,14 +167,15 @@ async fn restore_fids(
 
         let output = output.await?;
 
-        // TODO: handle termination conditions without defaulting to 0
-        let res = FidError {
-            fid: fid.fid.clone(),
-            data: fid.data.clone(),
-            errno: output.status.code().unwrap_or(0) as i16,
-        };
-        result.push(res);
-
+        if !output.status.success() {
+            // TODO: handle termination conditions without defaulting to 0
```

Comment:
This comment is possibly no longer needed (The above check will fail on a signal).